Makefile

```diff
@@ -76,6 +76,7 @@ else
 travis-release: goreleaser kopia-ui website
 	$(MAKE) test-all
 	$(MAKE) integration-tests
+	$(MAKE) robustness-tests
```

Comment:
Do we want to separate the actual robustness tests with the test for the robustness toolchain?
If so, it may be a good idea to rename the target. `tools-tests` (too generic) , `robustness-tools-tests`, other ?

Reply:
Yeah it's a good idea. I'll rename it.